content/en/monitors/create/types/error_tracking.md

```diff
@@ -0,0 +1,156 @@
+5. Configure the alerting grouping strategy (optional).
+   - Simple-Alert: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. If the query has a `group by` and you select **Simple-Alert**, you get one alert when one or multiple groups’ values breach the threshold. You may use this strategy to reduce notification noise.
+   - Multi-Alert: Multiple alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you can group a query by `@browser.name` to receive a separate alert for each browser when the number of errors is high.
+
+
+[1]: /real_user_monitoring/explorer/search/
+{{% /tab %}}
+
+{{% tab "New Issue" %}}
+
+As you expand your search filter, the issue details list above **Select the alerting condition** updates.
+
+1. Select or input a custom time period for an issue to be considered new after its first occurrence.
+2. Select **Web and Mobile Apps** or **Backend Services** and choose to monitor over a count or [measure][1]. 
+   - Monitor over a unique count based on the issue ID. 
+   - Monitor over a measure. If you select a measure, the monitor alerts over the numerical value of the RUM facet (similar to a metric monitor). Select an aggregation type (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
+3. Construct a search query using the same logic as a [RUM Explorer search][2] for the issues’ error occurrences.
+4. Configure the alerting grouping strategy (optional).
+   - Simple-Alert: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. If the query has a `group by` and you select **Simple-Alert**, you get one alert when one or multiple groups’ values breach the threshold. You may use this strategy to reduce notification noise.
+   - Multi-Alert: Multiple alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you can group a query by `@browser.name` to receive a separate alert for each browser when the number of errors is high.
+
+
+[1]: /real_user_monitoring/explorer/?tab=measures#setup-facets-measures
+[2]: /real_user_monitoring/explorer/search/
+{{% /tab %}} 
+{{< /tabs >}}
+
+### Set alert conditions
+
+#### Count
+
+{{< tabs >}}
+{{% tab "Error Occurrences" %}}
+
+Triggers when the metric is `above` or `above or equal to`.
+
+{{% /tab %}}
+{{% tab "Impacted Users" %}}
+
+Triggers when the number of impacted user emails is `above` or `above or equal to`.
+
+{{% /tab %}}
+{{% tab "Impacted Sessions" %}}
+
+Triggers when the number of impacted session IDs is `above` or `above or equal to`.
+
+{{% /tab %}} 
+{{< /tabs >}}
+
+#### New Issue
```

Comment:
Add a section about the time period that the issue is considered "new".
`Select time period Consider an issue as new up to x minutes after its very first error occurrence`

e.g.: "The Monitor will consider the issue as new for the selected time-period, after that it will turn back to green and stop alerting"

We should also point out that the selected threshold will only be evaluated in that selected timeframe, maybe with an example?

Reply:
Add a section that explains that the preview shows issue that would have been considered new in the selected bigger timeframe like past 24h or past week.

Reply:
To your first point about "new" issues, this info already exists in the first New Issue tab!